datascience/tables.py

```diff
@@ -151,7 +151,7 @@ def from_df(cls, df):
         """Convert a Pandas DataFrame into a Table."""
         t = cls()
         labels = df.columns
-        for label in df.columns:
+        for label in labels:
```

Comment:
I think I'd prefer a change that insteaddeletes the unused `labels = df.columns` line.  I don't see why this would make any difference in efficiency.

Reply:
I think it was left this way for readability - "df.columns" isn't as descriptive as "labels" for what it really is. Thoughts?

Reply:
OK, seems fine to me.